tests/config/functional/toml/issue_3181/top_level_list_of_disable.2.out

```diff
@@ -1,3 +0,0 @@
-************* Module {abspath}
-{relpath}:1:0: E0014: Out-of-place setting encountered in top level configuration-section 'max-line-length' : '120' (bad-configuration-section)
-{relpath}:1:0: E0014: Out-of-place setting encountered in top level configuration-section 'disable' : '['C0330']' (bad-configuration-section)
```

Comment:
We no longer raise this message because message can't be "out-of-place" with the `argparse` parsing. This only shows up now because I changed the functional tests to check `namespace` instead of `config`.